### PR TITLE
Implemented site navbar and mobile menu

### LINKED ISSUE #1: Implement Navbar
# Context

simian-navbar

# Acceptance Criteria

* Navbar functionality should be similar to this one https://slack.com/enterprise

### PR DESCRIPTION
**Pull request by TheMightyPenguin**
# Context

This was done by:

- Added markup and styling for navbar and alt-navbar
- Added markup and styling for mobile menu
- Added JS functions to handle navbar functionality
- Added JS functions to handle mobile menu functionality

Also refactor footer, no longer handled with flexbox so compatibility is increased.

This closes #1 

Also check page on the [staging site](https://simian-ci.firebaseapp.com/)

## Links

- [Neat backface-visibility trick](http://stackoverflow.com/questions/29303774/why-facebook-topbar-with-positionfixed-not-doing-repaint) this prevents repaint on navbar
- [Slack reference page](https://slack.com/enterprise)
- [Zeplin desktop design](https://app.zeplin.io/project.html#pid=586eaacfec55da7c2d67a2df&sid=58d442a75c718fae5988b30c)
- [Zeplin mobile design](https://app.zeplin.io/project.html#pid=586eaacfec55da7c2d67a2df&sid=587ac68562ad85054dcfaa4d&ref=slack)
- [Zeplin tablet design](https://app.zeplin.io/project.html#pid=586eaacfec55da7c2d67a2df&sid=58dd76421f2e8155af30c62e)

## Media

Navbar Desktop
Navbar Mobile

### CODE REVIEW

File: _sass/_components--navbar.scss
Comment (duranmla) [inline _sass/_components--navbar.scss]: always space after `,`

File: _sass/_layouts--global.scss
Comment (duranmla) [inline _sass/_layouts--global.scss]: We can remove this class  and use bootstrap's one

File: _sass/_objects--buttons.scss
Comment (duranmla) [inline _sass/_objects--buttons.scss]: I think so far you should know, and if it is a matter of asking let's wait for the answer before merge

File: _sass/_objects--menu-toggle.scss
Comment (duranmla) [inline _sass/_objects--menu-toggle.scss]: separate operators like `$bla * 10`

File: _sass/_objects--menu-toggle.scss
Comment (duranmla) [inline _sass/_objects--menu-toggle.scss:28]: why so many of this?

**Review by duranmla — CHANGES_REQUESTED** [review]
There are some minor stuff to clarify before merge this

File: _sass/_objects--menu-toggle.scss
Comment (TheMightyPenguin) [reply _sass/_objects--menu-toggle.scss:28]: That mixins adds the style necessary for displaying each line of the menu toggler! So its needed on the element itself and the pseudoelements

**Review by duranmla — APPROVED** [review]

**Pull request merged**
